### PR TITLE
Fix clash of gifs autoplaying breaking video embeds

### PR DESCRIPTION
**Pull request by schwers**
My gif autoplay patch broke video playing. But I didn't realize that or that it was patched. This attempts to make both of those coexist and fixes imgur.gifv's

### CODE REVIEW

**Comment by schwers** [comment]
:eyeglasses: @ajacksified @curioussavage

File: src/views/components/ListingContent.jsx [hunk 1]
```diff
@@ -132,6 +132,15 @@ class ListingContent extends BaseComponent {
     return null;
   }
 
+  useOurOwnPLayingOrPreviewing(oembed, url) {
```
Comment (umbrae) [inline src/views/components/ListingContent.jsx]: Capital L in Playing?

**Comment by curioussavage** [comment]
:fish: tested and code looks good.

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**